recipe/meta.yaml

```diff
@@ -23,7 +24,7 @@ requirements:
     - numpy
     - packaging
     - pandas
-    - python
+    - python >=3.6
```

Comment:
Also add to the python in host, I believe.

Reply:
Oh yes, good catch.

```suggestion
    - python >=3.6
```